modules/tinymce/src/plugins/table/test/ts/browser/ui/TableColumnHeaderUiTest.ts

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable mocha/no-exclusive-tests */
```

Comment:
```suggestion
```

We have to be very careful adding this, it shouldn't be needed to run the tests with `.only` either
- [ ] Remove eslint comment